src/components/features/letter-remind/TodoContainer/index.tsx

```diff
@@ -0,0 +1,65 @@
+import * as S from "./styled";
+import { useState } from "react";
+import Image from "next/image";
+import ToggleArrowButton from "@/src/components/common/Buttons/ToggleArrowButton";
+
+interface ObjectProps {
+  title: string;
+  date: string;
+  situation: string;
+  alarm: string;
+  content: string;
+  isAlarm: boolean;
+}
+interface Props {
+  todoObject: ObjectProps;
+}
```

Comment:
```suggestion
interface Props {
  todo: {
       title: string;
       date: string;
       situation: string;
       alarm: string;
       content: string;
       isAlarm: boolean;
    }
}
```

`ObjectProps` 가 외부에서 다시 사용되는게 아니라면 그냥 하나로 만들어도 좋을 것 같습니다!
이후 api 를 통해 투두를 받아오면 따로 `Todo` 라는 타입을 만들어서 여러 곳에 사용하게 될 것 같아서요!
`interface Props { todo: Todo }`

Reply:
오 넵 알겠습니다! 이 방법이 더 좋겠네요